android/pytorch_android/src/main/java/org/pytorch/IValue.java

```diff
@@ -84,54 +90,83 @@ public static IValue optionalNull() {
     return new IValue(TYPE_CODE_NULL);
   }
 
+  /**
+   * Creates new IValue instance of torchscript Tensor type.
+   */
   public static IValue tensor(Tensor tensor) {
     final IValue iv = new IValue(TYPE_CODE_TENSOR);
     iv.mData = tensor;
     return iv;
   }
 
+  /**
+   * Creates new IValue instance of torchscript bool type.
+   */
   public static IValue bool(boolean value) {
     final IValue iv = new IValue(TYPE_CODE_BOOL);
     iv.mData = value;
     return iv;
   }
 
+  /**
+   * Creates new IValue instance of torchscript int type.
+   */
   public static IValue long64(long value) {
     final IValue iv = new IValue(TYPE_CODE_LONG);
     iv.mData = value;
     return iv;
   }
 
+  /**
+   * Creates new IValue instance of torchscript float type.
+   */
   public static IValue double64(double value) {
     final IValue iv = new IValue(TYPE_CODE_DOUBLE);
     iv.mData = value;
     return iv;
   }
 
+  /**
+   * Creates new IValue instance of torchscript List[bool] type.
+   */
   public static IValue boolList(boolean... list) {
     final IValue iv = new IValue(TYPE_CODE_BOOL_LIST);
     iv.mData = list;
     return iv;
   }
 
+  /**
+   * Creates new IValue instance of torchscript List[int] type.
+   */
   public static IValue longList(long... list) {
     final IValue iv = new IValue(TYPE_CODE_LONG_LIST);
     iv.mData = list;
     return iv;
   }
 
+  /**
+   * Creates new IValue instance of torchscript List[float] type.
+   */
   public static IValue doubleList(double... list) {
     final IValue iv = new IValue(TYPE_CODE_DOUBLE_LIST);
     iv.mData = list;
     return iv;
   }
 
+  /**
+   * Creates new IValue instance of torchscript List[Tensor] type.
+   */
   public static IValue tensorList(Tensor... list) {
     final IValue iv = new IValue(TYPE_CODE_TENSOR_LIST);
     iv.mData = list;
     return iv;
   }
 
+  /**
+   * Creates new IValue instance of torchscript List[T] type, all elements must have the same,
```

Comment:
Creates a new IValue instance of torchscript List[T] type.  All elements must have the same type.